### ISSUE TITLE
read_group does not return members id and display name

### ISSUE THREAD

**Issue by adarsh-chauhan** [seratch/python-slack-scim#1]
Hi,

I was trying to use the library for read_group method however the response object does not contain member object attributes.
All seem to be none.

```
client = slack_scim.SCIMClient(xoxp-XXXXXXXXXX)
response = client.read_group('XXXXXXXXX')
```
```
response.__dict__
{'display_name': 'LDAP_slack-xxxxxxxx', 'id': 'XXXXXXXXX', 'members': [<slack_scim.v1.group.Member object at 0x1065b7730>, <slack_scim.v1.group.Member object at 0x1065b7670>, <slack_scim.v1.group.Member object at 0x1065b76a0>, <slack_scim.v1.group.Member object at 0x1065b7850>, <slack_scim.v1.group.Member object at 0x1065b77c0>, <slack_scim.v1.group.Member object at 0x1065b7820>, <slack_scim.v1.group.Member object at 0x1065b77f0>, <slack_scim.v1.group.Member object at 0x1065b78b0>, <slack_scim.v1.group.Member object at 0x1065b78e0>, <slack_scim.v1.group.Member object at 0x1065b7940>, <slack_scim.v1.group.Member object at 0x1065b7970>, <slack_scim.v1.group.Member object at 0x1065b7910>], 'meta': <slack_scim.v1.group.Meta object at 0x106594fd0>, 'schemas': ['urn:scim:schemas:core:1.0']}
```

```
res.members[0].__dict__
{'active': None, 'display_name': None, 'external_id': None, 'id': None, 'locale': None, 'meta': None, 'name': None, 'nick_name': None, 'password': None, 'preferred_language': None, 'profile_url': None, 'schemas': None, 'timezone': None, 'title': None, 'urn_scim_schemas_extension_enterprise_10': None, 'user_name': None, 'user_type': None}
```

The expected behaviour is that member's id and display_name should be populated as it was received in the response.

```
DEBUG:slack_scim.v1.client:*** SCIM API Response ***
GET https://api.slack.com/scim/v1/Groups/XXXXXX
200 OK
date: Thu, 30 Jul 2020 09:57:07 GMT
server: Apache
strict-transport-security: max-age=31536000; includeSubDomains; preload
referrer-policy: no-referrer
x-slack-backend: r
vary: Accept-Encoding
x-xss-protection: 0
x-frame-options: SAMEORIGIN
connection: close
transfer-encoding: chunked
content-type: application/json; charset=utf-8

{"schemas":["urn:scim:schemas:core:1.0"],"id":"XXXXXXX","displayName":"xxxxxxxx","members":[{"value":"XXXXX","display":"Michael Jackson"}],"meta":{"created":"2019-03-18T09:05:06-07:00","location":"https:\/\/api.slack.com\/scim\/v1\/Groups\/XXXXXX"}}
```

**Comment by seratch** [comment]
Thank you very much for taking the time to report this. As you pointed out, the generated classes are invalid. I will take a look at the issue very soon. 🙇

**Comment by seratch** [comment]
@adarsh-chauhan I'm sorry for the inconvenience you had yesterday. I've corrected the generated classes and published v1.1.0 to the PyPI registry. Please try the version out. Thanks again for reporting the issue!
